CHANGES.md

```diff
@@ -8,15 +8,21 @@
   - Add `is_split_allowed` to check if a store allows split. (#2175, @metanivek)
   - Add `add_volume` to allow creating new empty volume in lower layer. (#2188,
     @metanivek)
+  - Add a `behavior` function to the GC to check wether the GC will archive or
```

Comment:
I've usually seen standard British English spellings (eg, finalise vs finalize) in the project, but I don't know if this is an intentional style choice. @samoht any input on this?

Reply:
I actually wondered when choosing the name :p A quick grep shows a mixed usage but indeed more occurrences of `behaviour`

Reply:
Hmm, I don't have a strong opinion on this, so we can go with whichever you prefer. :smile:

Reply:
Went with `behaviour` :p

Reply:
The first version of Irmin was written at Cambridge - it inherited the British English spelling ;p 

I don't have a strong opinion, but I try to keep this consistent with the rest of the codebase (so I tend to continue to favour British English in comments).